conduct/index.md

```diff
@@ -0,0 +1,73 @@
+
+If a participant engages in unacceptable behavior, the organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event).
+
+## If You are Witness or Subject to Unacceptable Behavior
+
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a Maptime organizer as soon as possible: maptimesea [at] gmail [dot] com.
+
+Additionally, community organizers are available to help community participants engage with local law enforcement or to otherwise help those experiencing unacceptable behavior feel safe. In the context of in-person events, organizers will also provide escorts as desired by the person experiencing distress.
+
+## Addressing Grievances
+
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify Maptime HQ with a concise description of your grievance, and we will address it.
+
+## Scope
+
+We expect all community participants (contributors, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all community venues—online and in-person—as well as in all one-on-one communications pertaining to community business.
+
+## Contact
+
+maptimesea [at] gmail [dot] com
+
+## License and attribution
+
+This Code of Conduct is distributed under a Creative Commons Attribution-Share Alike license. Revision 2.0, adopted by the Stumptown Syndicate board on 10 January 2013. Posted 17 March 2013.
+
+## Additional List of Things that are Totally Not OK
+
+The following items are worth calling out as activities that are prohibited from Maptime:
+
+- **Selling your product**. Maptime is a place for co-learning. We often talk about how to use open source tools, some of which require a fee after a certain level of usage (such as CartoDB or Mapbox). However, using Maptime as an opportunity to sell or pitch your product is prohibited. We will not tolerate anyone – sponsors, students, or another participant – using this time for co-learning as a time to get people to buy your product.
+
+- **Seeking volunteers to build your product.** Are you looking for free labor for your idea? You have come to the wrong place.
+
+- **But wait, what if it’s an artist or nonprofit project and a lot of us really want to work on it?** Isn’t Maptime a great place to make that happen? Totally. If Maptime students decide that they want to work on a project for a good cause or an artist or someone who really doesn’t have any money, sure! Local Chapters should decide among themselves if they want to devote Maptime time to this project, or if the project needs to find its own meeting time. Seeking free consultation for your product or idea. We recommend hiring experts for this kind of thing.
```

Comment:
The line that reads 

> Seeking free consultation for your product or idea.

...is a sentence fragment. Here's a shot at a clarifying that sentence, and making clear that we would encourage the sharing of ideas and advice, just not the soliciting of free labor.

> Seeking free consultation for your product or idea is prohibited; we recommend hiring experts for this kind of thing. We do, however, encourage Maptimers to seek help on which open source tools might be applicable to personal projects.

Here's that as a suggestion to modify the paragraph:

```suggestion
 - **But wait, what if it’s an artist or nonprofit project and a lot of us really want to work on it?** Isn’t Maptime a great place to make that happen? Totally. If Maptime students decide that they want to work on a project for a good cause or an artist or someone who really doesn’t have any money, sure! Local Chapters should decide among themselves if they want to devote Maptime time to this project, or if the project needs to find its own meeting time. Seeking free consultation for your product or idea is prohibited; we recommend hiring experts for this kind of thing.  We do, however, encourage Maptimers to seek help on which open source tools might be applicable to personal projects.
```

Reply:
I guess we should clean that up further since it talks about local chapters, but we do not yet have local chapters of maptime sea =]